homeassistant/components/lutron/fan.py

```diff
@@ -0,0 +1,123 @@
+"""Support for Lutron fans."""
+import logging
+
+from homeassistant.components.fan import (SPEED_LOW,
+                                          SPEED_MEDIUM,
+                                          SPEED_HIGH,
+                                          SPEED_OFF,
+                                          SUPPORT_SET_SPEED,
+                                          FanEntity)
+from . import LUTRON_CONTROLLER, LUTRON_DEVICES, LutronDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+SPEED_MEDIUM_HIGH = 'medium_high'
+SPEED_MAPPING = {
+    SPEED_OFF: 0,
+    SPEED_LOW: 25,
+    SPEED_MEDIUM: 50,
+    SPEED_MEDIUM_HIGH: 75,
```

Comment:
Only speeds in the base fan component are allowed to be returned in the speed list or accepted as set speed argument.

Please remove the `SPEED_MEDIUM_HIGH`.

Reply:
Are you suggesting that people won't be able to set their fans to 75% then? If that is the case, maybe leaving them to be discovered as a dimmer would make more sense.

Reply:
We can map the available speed constants as best we can, but we can't introduce new ones.

Reply:
Right on, I understand that there can't be special cases for 1 product, makes sense to me. Do you have a feeling on what we should do with this? The fan switches physically have 4 LEDs for their 25,50,75,100 percent states, I feel like it'd be doing the integration a disservice to remove the ability to set it to 75% (especially because people that are using it can already do it right now because they're treated as a regular dimmer). Thanks for your input, I really appreciate it!

Reply:
Maybe don't change anything until we have a better interface for fans?

Reply:
I feel like that's the right move for now. I'll close out this PR and stash it away for later :D 
Thanks Martin!